nexus3/manifest.yaml

```diff
@@ -0,0 +1,13 @@
+---
+name: nexus3
+title: "Sonatype nexus repository oss"
+version: "3.30.1"
+maintainer: kamesh.sampath@hotmail.com
+description: The free artifact repository with universal format support.
+url: https://www.sonatype.com/products/repository-oss
+category: ci_cd
+plans:
```

Comment:
if only one plan, then why use plan? you can directly use 15 !

Reply:
not sure I got what you mean here .. can you please give more details?

Reply:
you can remove plan and just use the values, Plan us used in cases where you want to give multiple options

Reply:
Like Amit said, plans are to give multiple options to the user, but if you are only using one plan then you can directly use that value instead of plan

Reply:
@saiyam1814  @DoNnMyTh  do you mean like 

```
...
configuration:
  RAM:
    value: 4
  CPU:
    value: 2
  VOLUME_SIZE:
    value: 15
```

Reply:
Please see how the plans are used for Mongodb or others, its for giving selection to the users, but if you have only one plan then why to use it ? directly use the value

Reply:
can you remove plans from this PR as well if you are not using this ?

Reply:
yep done that @saiyam1814 . thank you